packages/terra-toggler/src/Toggler.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import AnimateHeight from 'react-animate-height';
+import 'terra-base/lib/baseStyles';
+import './Toggler.scss';
+
+const propTypes = {
+  /**
+   * Content in the body of the toggler component that will be expanded or collapsed
+   */
+  children: PropTypes.node.isRequired,
+  /**
+   * When set, will animate the toggler component as it is expanded or collapsed
+   */
+  isAnimated: PropTypes.bool,
+  /**
+   * Used to expand or collapse toggler content
+   */
+  isOpen: PropTypes.bool,
+};
+
+const defaultProps = {
+  children: null,
+  isAnimated: false,
+  isOpen: false,
+};
+
+const Toggler = ({ isAnimated, isOpen, children, ...customProps }) => {
+  const attributes = Object.assign({}, customProps);
+  const TogglerClassNames = classNames([
+    'terra-Toggler',
+    attributes.className,
+  ]);
+  const height = isOpen ? 'auto' : '0';
+  let body;
+
+  if (isAnimated) {
+    body = (
+      <AnimateHeight
+        duration={250}
+        height={height}
+      >
+        {children}
+      </AnimateHeight>
+    );
+  } else {
+    body = (
+      isOpen && children
+    );
+  }
+
+  return (
+    <div {...attributes} className={TogglerClassNames} aria-hidden={!isOpen}>
+      {body}
```

Comment:
if isOpened = false, we should return null. 

If a focusable item is in the body, we can still tab to it even though it is hidden